scripts/measures/import-qcdr-measures.js

```diff
@@ -93,7 +93,7 @@ const addMultiPerformanceRateDetails = function(newMeasure, record, qcdrStrataNa
   const strataNames = fs.readFileSync(path.join(__dirname, qcdrStrataNamesDataPath), 'utf8');
   const qcdrStrataNames = JSON.parse(strataNames);
 
-  newMeasure['metricType'] = 'multiPerformanceRate';
+  newMeasure['metricType'] = 'registryMultiPerformanceRate';
```

Comment:
this is related to/necessary for a previous pr: https://github.com/CMSgov/qpp-measures-data/pull/99